src/recipient_dashboard/templates/recipient_dashboard/dashboard.html

```diff
@@ -29,9 +29,9 @@ <h2 class="subtitle mt-3">List of Available Donations:</h2>
                                                href="https://www.google.com/maps/dir/?api=1&destination={{ donation.organization_id.address|urlencode }}"
                                                target="_blank"
                                                rel="noopener noreferrer">
-                                                {{ donation.organization_id.organization_name }}
+                                                {{ donation.organization.organization_name }}
```

Comment:
FYI (no action required) - I don't think this will have merge conflicts, but I think we had talked about me handling this (and I opened a PR for it here: https://github.com/gcivil-nyu-org/wed-fall24-team5/pull/103) ! 

IMO this change should probably be in its own PR just to keep the scope of a PR centered

Reply:
Ahh cool, totally agree to your point!